apps/discord-bot/package.json

```diff
@@ -13,6 +13,7 @@
 		"@answeroverflow/db-mock": "workspace:^",
 		"@answeroverflow/discordjs-react": "^0.0.36",
 		"@answeroverflow/discordjs-utils": "workspace:^",
+		"@answeroverflow/elastic-types": "workspace:^",
```

Comment:
why did we add elastic types to discord bot

Reply:
make db export these types - import directly from db now